src/web/mjs/engine/Storage.mjs

```diff
@@ -779,18 +779,20 @@ export default class Storage {
      */
     sanatizePath(path) {
         if (this.platform.indexOf('win') === 0) {
-            // TODO: max. 260 characters per path
+            // Replace forbidden characters for Windows
```

Comment:
Comment is redundant and can be removed, it can easily be inferred from the code itself.
Other places as well in this file.

Reply:
Got it